perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/UsersManagerBlImpl.java

```diff
@@ -614,6 +616,12 @@ private void deleteUser(PerunSession sess, User user, boolean forceDelete, boole
 			}
 		}
 
+		//Remove all user's consents
+		for (Consent consent : getPerunBl().getConsentsManagerBl().getConsentsForUser(sess, user.getId())) {
+			getPerunBl().getConsentsManagerBl().deleteConsent(sess, consent);
```

Comment:
I think it would be better to catch the ConsentNotExists exception here, ignore it (we want to delete the consent anyway), and omit it from the signature. Because right now, it is caught in the other method and rethrown as InternalErrorException, I think it would be better to catch it here and skip it, so the whole method doesn't fail.